posts/inside-rust/2023-12-08-cargo-cache-cleaning.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "Cargo cache cleaning"
+author: Eric Huss
+team: The Cargo Team <https://www.rust-lang.org/governance/teams/dev-tools#cargo>
+---
+
+Cargo has recently gained an unstable feature on the nightly channel (starting with nightly-2023-11-17) to perform automatic cleaning of cache content within Cargo's home directory.
+This post includes a description of what this means, a call for help, and an outline of what is planned for the future.
```

Comment:
> an outline of what is planned for the future.

This makes it sound like there should be a section but I'm not seeing it.  The reason this is important is that one of the first questions people will likely ask is about `target/` as that is what most balloons in size.

Reply:
This is the last section, which talks about `target` cleaning. I have adjusted things a little to make this outline a little clearer. Does that help?

Reply:
Sure, I went ahead and put a mention in the introduction.